src/Identity/EntityFrameworkCore/test/EF.Test/CustomPocoTest.cs

```diff
@@ -151,9 +136,6 @@ public async Task CanUpdateNameInt()
         }
 
         [ConditionalFact]
```

Comment:
I think these can all become `[Fact]` again if all the conditions have been removed

Reply:
I'll do it in another PR.